docs/manuals/studio/config/elements.md

```diff
@@ -128,6 +137,10 @@ element.<br/>
 Note that you cannot set the identifier of a new Pipeline element to one
 already used by another.
 
+Default values are provided for the essential properties of the new Pipeline configuration
+element: *tasks* is set to an empty list.<br/>
+These properties are displayed in the DETAILS section of the "Taipy Configs" panel.
```

Comment:
this property

Reply:
Right. The wording is kind of bad. I wanted to say... all the element properties.
I'll fix that.
Thanks!